terraform/app_service.tf

```diff
@@ -98,7 +98,7 @@ resource "azurerm_windows_web_app_slot" "hro-api" {
     LEASEHOLD_CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID = "@Microsoft.KeyVault(SecretUri=${azurerm_key_vault_secret.leasehold-confirmation-email-notify-template-id.id})"
     LEASEHOLD_CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID   = "@Microsoft.KeyVault(SecretUri=${azurerm_key_vault_secret.leasehold-confirmation-sms-notify-template-id.id})"
     LEASEHOLD_INTERNAL_EMAIL_NOTIFY_TEMPLATE_ID     = "@Microsoft.KeyVault(SecretUri=${azurerm_key_vault_secret.leasehold-internal-email-notify-template-id.id})"
-    CANCELLATION_INTERNAL_NOTIFY_TEMPLATE_ID        = "@Microsoft.KeyVault(SecretUri=${azurerm_key_vault_secret.cancellation-internal-notify-template-id.id})"
+    CANCELLATION_INTERNAL_EMAIL_NOTIFY_TEMPLATE_ID  = "@Microsoft.KeyVault(SecretUri=${azurerm_key_vault_secret.cancellation-internal-notify-template-id.id})"
```

Comment:
In [key_vault.tf](https://github.com/Newarkandsherwood/housing-repairs-online-api/blob/922538d9cf22006acbe0127ae75dca03d6a63d2c/terraform/key_vault.tf), there's a slight inconsistency too. The naming at the start of the declaration is still cancellation-internal-notify-template-id. I'm not sure if it's this or the name property that's used for referencing it but it would be good to make them the same as per other key vault declarations in the file.

resource "azurerm_key_vault_secret" "cancellation-internal-notify-template-id" {
  name         = "cancellation-internal-email-notify-template-id"
  value        = var.cancellation_internal_email_notify_template_id
  key_vault_id = azurerm_key_vault.hro-repairs-api-key-vault.id
}

Reply:
Good catch!

I've renamed those ones too now.